CONTRIBUTING.md

```diff
@@ -108,6 +108,25 @@ to make that happen.
 
 Happy developing!
 
+### Using DummyAuthenticator & SimpleSpawner
+
+The default JupyterHub [authenticator](https://jupyterhub.readthedocs.io/en/stable/reference/authenticators.html#the-default-pam-authenticator)
+& [spawner](https://jupyterhub.readthedocs.io/en/stable/api/spawner.html#localprocessspawner)
+require your system to have user accounts for each user you want to log in to
+JupyterHub as.
+
+[DummyAuthenticator](https://github.com/jupyterhub/dummyauthenticator) allows
+you to log in with any username & password.
+[SimpleSpawner](https://github.com/jupyterhub/simplespawner) lets you start
```

Comment:
While, SimpleSpawner starts servers without having to create a unix user for each JupyterHub user. ~~Together...JupyterHub~~